### DISCUSSION TITLE
Any live example?

### DISCUSSION THREAD

**Discussion by iniznet** [Accudio/tailwind-smart-fluid#6] [General]
Honestly, sometimes I have a little bit of issue with tailwind when coming to responsive spacing, this might be able to solve my issue, but before going to proceed with my own experiment, is there any live example that I can check?

**Comment by Accudio** [comment]
I haven't put together any demos or such, but you can check out [utilityinnovation.com](https://utilityinnovation.com/) for an example of a site that uses the plugin. It's not a great example but all I've got live at the moment.
